schema/visibility/versioned/v0.2/reduce_open_workflow_tombstones.cql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE cadence_visibility.open_executions WITH gc_grace_seconds=60 AND compaction = {
+  'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy',
+  'tombstone_threshold': 0.6
```

Comment:
0.6 is pretty high compared with default value 0.2. Any special reason put 0.6 here?

Reply:
Not exactly ;) I wanted to test it out to see what performance is like. There are tradeoffs for both, if threshold is too low and at the rate of tombstones we have, compaction delays can in turn cause slow reads. If threshold is too high, the number of rows being scanned is too high also causing read delays. Was trying to find a balance between the two. I am still running some tests before finalizing on this number.

Reply:
Awesome, Thanks. Would be great if you could share the test results later.

Reply:
Tried out on my laptop and staging. Works out fine on staging. On my laptop (where TPS was higher), I saw quite a few tombstone warnings. Lowering this to 0.4 performed better. Changed it to 0.4